pkg/collector/corechecks/oracle-dbm/oracle.go

```diff
@@ -180,6 +187,18 @@ func (c *Check) Connect() (*sqlx.DB, error) {
 		c.tags = append(c.tags, fmt.Sprintf("host:%s", c.dbHostname), fmt.Sprintf("oracle_version:%s", c.dbVersion))
 	}
 
+	if c.filePath == "" {
+		r := db.QueryRow("SELECT SUBSTR(name, 1, 10) path FROM v$datafile WHERE rownum = 1")
```

Comment:
`WHERE rownum = 1`

This condition is slightly concerning due to my unfamiliarity with this view. This will always be consistently `1` regardless of version and customer instance? Is this the most canonical way to know if the database is RDS or not?

Reply:
Yes, it's always 1. rownum isn't part of this view; it's rather a pseudo-column which contains the position of a row in the result set. It shouldn't be confused with the rowid which is effectively the row address and consists of file_id,block_id and the position of the row within the block. This clause is 100% safe.